phrosty/imagesubtraction.py

```diff
@@ -532,54 +567,77 @@ def swarp_coadd_img(imgpath_list,refpath,out_name,out_path=output_files_rootdir,
         else:
             imgpaths.append(p)
 
-    coadd_savedir = os.path.join(out_path,'coadd')
+    if psf:
+        subdir = 'coadd_psf'
+    elif not psf:
```

Comment:
If "psf" is really just about specifying the subdir, I would suggest that `subdir` be the argument to the `swarp_coadd_img` function rather than psf.

Reply:
In general, there's a lot of logic that's embedded in filepath choices within functions.  I think we want to move in a general direction of filepaths being explicitly passed in and out, or are at least being in keyword parameters that are thus visible in the documentation for a function and overridable.